SUMMARY.md

```diff
@@ -147,8 +147,11 @@
     * [storeConfiguration](collector/remote-control/apis/storeconfiguration.md)
     * [getFromStore](collector/remote-control/apis/getfromstore.md)
   * [Sources](collector/remote-control/sources.md)
-  * [Requests](collector/remote-control/requests.md)
+  * [Witsml Requests](collector/remote-control/witsml-requests.md)
+  * [Opc Requests](collector/remote-control/opc-requests.md)
+  * [Object Explorer](collector/remote-control/object-explorer.md)
```

Comment:
```suggestion
  * [WITSML Object Explorer](collector/remote-control/object-explorer.md)
```

Reply:
But this will demand the rename of the `remote-control/object-explorer.md` to `remote-control/witsml-object-explorer.md` to keep the gitbook consistency